crates/crypto/merkle/src/hash.rs

```diff
@@ -0,0 +1,6 @@
+use alloy_primitives::B256;
+
+/// Common hashing function for Merkle trees.
+pub(crate) fn hash_concat(h1: &[u8], h2: &[u8]) -> B256 {
```

Comment:
```suggestion
pub fn hash_concat(h1: &[u8], h2: &[u8]) -> B256 {
```

same reason as before